bitherj/src/main/java/net/bither/bitherj/core/TxBuilder.java

```diff
@@ -598,6 +598,10 @@ public int compare(Out out1, Out out2) {
 
             if (lastCalculatedSize >= 0) {
                 fees = lastCalculatedSize * (Utils.getFeeBase() / 1000);
+                long residual = fees % Utils.getFeePrecision();
```

Comment:
There are multiple places where the fees are set in this file. Except for the BuildBCCTx functions which are irrelevant to us, it is best to handle the rounding uniformly through a common call. 
Look for other places similar to line 600.
You can refer to Win's pull request earlier: https://github.com/primecoin/primerj/pull/3

Reply:
I have corrected this, and created a new pull request.